src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php

```diff
@@ -129,9 +129,19 @@ protected function describeContainerDefinition(Definition $definition, array $op
     /**
      * {@inheritdoc}
      */
-    protected function describeContainerAlias(Alias $alias, array $options = array())
+    protected function describeContainerAlias(Alias $alias, array $options = array(), ContainerBuilder $builder = null)
     {
-        $this->writeData($this->getContainerAliasData($alias), $options);
+        if (!isset($options['id']) || !$builder) {
+            return $this->writeData($this->getContainerAliasData($alias), $options);
+        }
+
+        $serviceId = $options['id'];
+
+        do {
+            $serviceId = (string) $builder->getAlias($serviceId);
```

Comment:
would it be possible to write data for each alias as well? So that if you have `alias1 -> alias2 -> service` you see something along the lines of:

```
- alias1 is an alias for alias2
- alias2 is an alias for service
- <service description>
```